framework/src/controller/ws/ws_server.ts

```diff
@@ -41,11 +41,11 @@ export class WSServer {
 		this._logger = options.logger;
 	}
 
-	public start(
-		registeredEvents: ReadonlyArray<string>,
-		messageHandler: WSMessageHandler,
-	): WebSocket.Server {
-		this._registeredEvents = registeredEvents;
+	public registerAllowedEvent(events: string[]) {
+		this._registeredEvents.push(...events);
```

Comment:
removed registration in the start, and being registered separately due to the initialization order